### PR TITLE
I have designed the Desktop portfolio version

### PR DESCRIPTION
**Pull request by migcm06**
* On this occasion, I used queries to implement the changes for larger screens.
* I have changed some background images and styles too.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -491,3 +493,338 @@ textarea.input-textarea::placeholder {
...
+  }
+
+  .fr {
+    margin-left: 19%;
+  }
+
+  .sk {
+    margin-left: -64%;
+  }
+
```
Comment (zieeco) [inline styles.css:1082]: - Hey, please, let's kindly fix the placement of these headers to be aligned correctly with the tags as seen in the images below to improve your design:


*your design*
image

*Figma design*
image

File: styles.css [hunk 2]
```diff
@@ -491,3 +493,338 @@ textarea.input-textarea::placeholder {
...
+    width: 116%;
+    height: 54vh;
+    margin-left: -1.2%;
+  }
+
+  .image-container {
+    display: flex;
+    width: 100%;
+    margin-bottom: 15%;
+  }
```
Comment (zieeco) [inline styles.css:959]: - Hi, please try to reduce the margin-bottom as this is making your design have too much space at the bottom making your design not consistent as compared to the original design:

*your design*
image


*Figma design*
image

File: styles.css [hunk 3]
```diff
@@ -491,3 +493,338 @@ textarea.input-textarea::placeholder {
...
+
+  .icon {
+    padding-bottom: 5%;
+  }
+
+  h1 {
+    position: absolute;
+    top: -39%;
+    left: 3%;
+  }
```
Comment (zieeco) [inline styles.css:910]: - Hey, let's also control the space at the bottom of the header here to improve our design. the space here is too much compared to the original design.

*Figma design*
image


*Your design*
image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @migcm06,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice using media query. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @migcm06 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**